R/tt_pos_and_access.R

```diff
@@ -291,7 +319,7 @@ setMethod("tt_at_path<-", c(tt = "VTableTree", value = "VTableTree"),
 })
 
 ## this one removes the child at path from the parents list of children,
-## becuase thats how lists behave.
+## because that's how lists behave.
```

Comment:
it is always better not have contraptions in written form (I know that it was as such already), but it is better to have `that is` in written form